Makefile

```diff
@@ -59,8 +59,16 @@ lint:
 	@echo "checking lint"
 	@./.tool/lint
 
-test:
-	go test -v -race -cover $(shell go list ./... | grep -v /vendor/)
+.PHONY: .gofmt .gotest
+
+PACKAGES = $(shell go list ./... | grep -v /vendor/)
+test: .gofmt .gotest
```

Comment:
I want to do `fmt` detection for the format, so I think it belongs to test, and I can also add `gofmt `commands in the makefile  to make formatting adjustments.